fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -1276,6 +1276,9 @@ ACTOR Future<Void> commitBatch(
 		ASSERT(p.second.isReady());
 	}
 
+	// Let master know this commit version so that every other proxy can know.
+	wait(self->master.reportLiveCommittedVersion.getReply(ReportRawCommittedVersionRequest(commitVersion, lockedAfter, metadataVersionAfter), TaskPriority::ProxyMasterVersionReply));
```

Comment:
This introduces a rt latency even if we choose not to get GRV from master.

Reply:
Done.